app/src/main/java/protect/card_locker/BarcodeImageWriterTask.java

```diff
@@ -120,6 +120,8 @@ private String getFallbackString(CatimaBarcode format) {
                 return "C0C";
             case CODE_39:
                 return "CODE_39";
+            case CODE_93:
+                return "CODE_93";
```

Comment:
`getFallbackString` is used in case the text the user entered can't generate a valid barcode. But `CODE_93` is not valid text in the Code 93 barcode:

![Screenshot_20211208_192906](https://user-images.githubusercontent.com/1885159/145263723-f036c495-a9d1-4a91-a5b0-e0cb10679d17.png)
(Checked on random site: https://barcode.tec-it.com/en/Code93?data=CODE_93)

Reply:
I just tested it in Catima and it is a valid code... At least it generates a code.
And I thought this string is also usefull to generate a preview when entering a manual codebar.

Reply:
The reason it might not work with your generator website is because there is also the code_93 extended format, which is maybe compatible with the "CODE_93" string :)

Reply:
Here a Code-93 extended bar code with "CODE_93":
![image](https://user-images.githubusercontent.com/66519591/145266118-b25866c1-e5d4-4d02-86fe-9f16b72e8ff3.png)